fingerprint/infra/bio-sdk/src/main/java/com/simprints/fingerprint/infra/biosdk/BioSdkResolverUseCase.kt

```diff
@@ -0,0 +1,34 @@
+package com.simprints.fingerprint.infra.biosdk
+
+import com.simprints.fingerprint.infra.biosdkimpl.SimprintsSdk
+import com.simprints.fingerprint.infra.necsdkimpl.NecSdk
+import com.simprints.infra.config.store.ConfigRepository
+import com.simprints.infra.config.store.models.FingerprintConfiguration
+import javax.inject.Inject
+
+class BioSdkResolverUseCase @Inject constructor(
```

Comment:
I usually suggest call use cases as if they were functions because on the callers side it will be used as one. (e.g. `ResolveBioSdkUseCase`).